app/src/main/java/org/stepik/android/view/step_quiz_fullscreen_code/ui/dialog/CodeStepQuizFullScreenDialogFragment.kt

```diff
@@ -149,13 +228,81 @@ class CodeStepQuizFullScreenDialogFragment : DialogFragment(), ChangeCodeLanguag
 
         codeLayoutDelegate.setLanguage(lang, code)
         codeLayoutDelegate.setDetailsContentData(lang)
-        fullScreenCodeViewPager.setCurrentItem(1, false)
+        fullScreenCodeViewPager.setCurrentItem(CODE_TAB, false)
 
         codeSubmitButton.setOnClickListener {
             (parentFragment as? Callback)
                 ?.onSyncCodeStateWithParent(lang, codeLayout.text.toString(), onSubmitClicked = true)
             dismiss()
         }
+
+        val inputSamples = stepWrapper
```

Comment:
let move all "run code" code to delegate, make it implement `StepQuizRunCodeView` and attach it to presenter